packages/venia-ui/lib/components/SearchPage/searchPage.js

```diff
@@ -194,6 +190,34 @@ const SearchPage = props => {
         .filter(Boolean)
         .join(' - ');
 
+    let areaLiveVal = 'off';
+    const productSortRoot = document.getElementById('ProductSort-root');
+    const filterSidebarRoot = document.getElementById('FilterSidebar-root');
+
+    if (productSortRoot) {
+        productSortRoot.setAttribute('aria-hidden', 'true');
+    }
+
+    if (filterSidebarRoot) {
+        filterSidebarRoot.setAttribute('aria-hidden', 'true');
+    }
+
+    if (productsCount > 0) {
+        areaLiveVal = 'polite';
+
+        if (productSortRoot) {
+            setTimeout(function() {
+                productSortRoot.setAttribute('aria-hidden', 'false');
+            }, 15000);
```

Comment:
this should be moved into the `ProductSort` component and done in a way such that a new timeout isn't scheduled everytime the component renders.

see: https://www.joshwcomeau.com/snippets/react-hooks/use-timeout/